gossip/src/crds_gossip_push.rs

```diff
@@ -157,6 +157,12 @@ impl CrdsGossipPush {
                         received_cache.record(origin, from, /*num_dups:*/ usize::MAX);
                         self.num_old.fetch_add(1, Ordering::Relaxed);
                     }
+                    Err(CrdsError::RedundantPull) => {
```

Comment:
If we are not distinguishing between `CrdsError::DuplicatePush(0)` and `CrdsError::RedundantPull` maybe we don't need to add `CrdsError::RedundantPull` anyways.
The `0` in `CrdsError::DuplicatePush(0)` should indicate this was received from push after it was upserted from pull-responses.

Reply:
oh you are right.